Campus-iOS/LoginComponent/ViewModel/TokenPermissionsViewModel.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  TokenPermissionsViewModel.swift
+//  Campus-iOS
+//
+//  Created by David Lin on 14.10.22.
+//
+
+import Foundation
+
+@MainActor
+class TokenPermissionsViewModel: ObservableObject {
+    
+    @Published var states: [PermissionType: State] = [
+            .grades: .na,
+            .calendar: .na,
+            .lectures: .na,
+            .tuitionFees: .na,
+            .identification: .na]
```

Comment:
Move the last array bracket into a sperate line

Reply:
✅